content/events/2019/xonotic_lan/contents.lr

```diff
@@ -0,0 +1,84 @@
+title: Xonotic LAN
+---
+body:
+
+# Game Night! - Xonotic LAN
+
+<br />
+
+#### Friday, November 15th
+
+##### EN 2036 (CS Lab, 6 to 8pm)
+
+_No registration required, just come on down!_
+
+<br />
+
+#### What is Xonotic?
+
+Xonotic is a modern yet authentic Quake; it runs on the 'DarkPlaces engine', which was initially based on the Quake 1 engine.
+
+As explained by: https://www.xonotic.org/
+
+_"Xonotic is an addictive, arena-style first person shooter with crisp movement and a wide array of weapons. It combines intuitive mechanics with in-your-face action to elevate your heart rate. Xonotic is and will always be free-to-play. It is available under the copyleft-style GPLv3+ license."_
+
+<br />
+
+#### How do I install it?
+
+## Installing on Labnet (MUN Desktops)
+
+If you want to use the desktops at the university to play Xonotic, there's a quicker way to download the game files.
+
+```bash
+# head to your home dir, and make a directory named 'games'
+cd ~
+mkdir games
+cd games
+
+# copy the xonotic zip to said ~/games folder, and unzip it 
+cp ~csclub/games/xonotic-0.8.2.zip .
```

Comment:
Have we confirmed that this shorthand works on labnet?

Reply:
yep